### PR TITLE
Main screen

### PR DESCRIPTION
**Pull request by Ksondzyk**

### CODE REVIEW

File: style/_base.scss [hunk 1]
```diff
@@ -1,35 +1,210 @@
...
+  box-sizing: inherit;
 }
-* {
-  clear: both;
+body {
+  font-family: Helvetica Neue LT Std;
 }
+// body * + * {
+//   margin-top: 1.5em;
+// }
```
Comment (layner95) [inline style/_base.scss]: delete comments

File: style/_base.scss [hunk 2]
```diff
@@ -1,35 +1,210 @@
...
 
 html {
-  box-sizing: border-box;
-
   font-size: 62.5%; // 1rem = 10px, 10px/16px = 62.5%
 
   // @media only screen and (max-width: $bp-large) {
   // font-size: 50%;
   // }
 }
```
Comment (layner95) [inline style/_base.scss]: delete comments and prettify your  code

File: style/_base.scss [hunk 3]
```diff
@@ -1,35 +1,210 @@
...
-li {
-  margin-left: 0;
-  padding-left: 0;
-  list-style-type: none;
-}
+// li {
+//   margin-left: 0;
+//   padding-left: 0;
+//   list-style-type: none;
+// }
```
Comment (layner95) [inline style/_base.scss]: Remove comments

File: style/_base.scss [hunk 4]
```diff
@@ -1,35 +1,210 @@
...
 .header-box {
   max-width: 100%;
   background-color: $colorcopper;
+  margin: 0 auto;
+  padding: 0 3em 1.5em;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  .menu-box {
```
Comment (layner95) [inline style/_base.scss:26]: add empty line

File: style/_base.scss [hunk 5]
```diff
@@ -1,35 +1,210 @@
...
+      background-color: #dab4af;
+    }
+    .block-three {
+      flex: 1;
+      display: flex;
+      background-color: #d3dae2;
+    }
+  }
+}
+.box-three {
```
Comment (layner95) [inline style/_base.scss]: add empty line

File: style/_base.scss [hunk 6]
```diff
@@ -1,35 +1,210 @@
...
-.menu-box {
-  max-width: 96vw;
+.box-two {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  &_main {
+    display: flex;
+    width: 100vw;
+    .block-one {
```
Comment (layner95) [inline style/_base.scss]: never use names like "block 1", "block 2"  rename this for example content or info

File: style/_base.scss [hunk 7]
```diff
@@ -1,35 +1,210 @@
...
+      flex: 1;
+      display: flex;
+      background-color: #b9c8cb;
+    }
+    .block-two {
+      flex: 1;
+      display: flex;
+      background-color: #dab4af;
```
Comment (layner95) [inline style/_base.scss]: use variables for colors

File: style/_base.scss [hunk 8]
```diff
@@ -1,35 +1,210 @@
...
+    }
+    .block-three {
+      flex: 1;
+      display: flex;
+      background-color: #d3dae2;
```
Comment (layner95) [inline style/_base.scss]: variables

File: style/_base.scss [hunk 9]
```diff
@@ -1,35 +1,210 @@
...
+    }
+  }
+}
+.box-three {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  padding: 0 3em 1.5em;
+  background-color: #c9cb7b;
```
Comment (layner95) [inline style/_base.scss]: variables

File: css/main.css [hunk 10]
```diff
@@ -2,32 +2,202 @@
...
+      background-color: #c9cb7b; }
+      .box-three_main .button-btn:hover {
+        max-width: 350px;
+        background-color: #c9cb7b; }
 
 a {
   text-decoration: none; }
 
+div {
+  font-family: 'Courier New', Courier, monospace; }
```
Comment (mariiaSynoverska) [inline css/main.css]: use global font value for project

File: style/_batton.scss [hunk 11]
```diff
@@ -1,3 +1,7 @@
 a {
   text-decoration: none;
 }
+
+div {
```
Comment (layner95) [inline style/_batton.scss]: don't use native elements. Change it to class

File: style/main.scss [hunk 12]
```diff
@@ -2,3 +2,4 @@
 @import "_batton.scss";
```
Comment (layner95) [inline style/main.scss:2]: rename to "_button.scss"

**Review by layner95 — CHANGES_REQUESTED** [review]

File: css/main.css [hunk 13]
```diff
@@ -2,32 +2,202 @@
...
+
 @font-face {
   font-family: "HelveticaNeueRegular";
   src: url("../assets/fonts/HelveticaNeueLTStd-It.otf"); }
```
Comment (mariiaSynoverska) [inline css/main.css:386]: move to the font.scss

**Pull request merged**